packages/fast-components-react-base/src/media/Media.props.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+
+/**
+ * Media HTML tags
+ */
+export enum MediaHTMLTags {
+    img = "img",
+    picture = "picture"
+}
+
+export interface IMediaProps {
+    /**
+     * The HTML alt attribute value is important for overall accessibility, providing a textual 
+     * alternative to non-text content 
+     */
+    alt: string;
+
+    /**
+     * The HTML itemScope attribute defines the scope of associated metadata and is used for associating the 
+     * element context with definitions on schema.org (important for seach engine optimization)
+     */
+    itemScope?: boolean;
+
+    /**
+     * The round image option
+     */
+    round?: boolean;
+
+    /**
+     * The HTML src attribute for the image element (specifying src will chose the image element rather than the picture element)
+     */
+    src?: string;
+
+    /**
+     * The HTML srcSet attribute for the image element
+     */
+    srcSet?: string;
+
+    /**
+     * The pictureSrcSet attribute for the picture element
+     */
+    pictureSrcSet?: string;
+
+    /**
+     * The HTML sizes attribute for the image element
```

Comment:
this prop makes sense for the <img> to me as a 1:1 mapping, but not as much with picture. This is defining size based on viewport width, which is essentially what we are wanting to do with the Picture element media query.

Reply:
We could pass one string but the standard <img /> srcset passes in a pixel value suffixed with "w" for width. 

```
<img 
   srcset="quilt_3/large.jpg  1240w, 
           quilt_3/medium.jpg  620w,
           quilt_3/small.jpg   310w"
   sizes="(min-width: 41.25em) 38.75em,
          calc(100vw - 2.5em)"
   src="quilt_3/medium.jpg"
   alt="A crazy quilt whose irregular fabric scraps are fit into a lattice of diamonds." />
```

It can also handle things like ratios:
`"<img src="standard.jpg" srcset="retina.jpg 2x, super-retina.jpg 3x" />"
`
I think keeping them separate removes any confusion about what element will be rendered. Here is more about using srcSets:
http://alistapart.com/article/responsive-images-in-practice